specification/personal-demographics.yaml

```diff
@@ -1188,12 +1205,9 @@ paths:
         However, historically this was constrained only by the integration requirements and was not enforced so theoretically more than one can exist for a patient when retrieving.
         Where multiple instances already exist for the patient it is not expected that local systems should manage those, but should choose the most appropriate one to maintain (for example by examining period dates).
 
-        A `home` address is the patient's main residential address and should normally be used for all clinical and demographic purposes, including clinical and appointment correspondence.
-        However additionally, `temp` and `billing` addresses may be provided by a patient when there is a requirement to record an alternative location for the purposes of unbroken care.
-        When sending correspondence to a patient:
-        *	a present and valid `billing` address may take precedence over `home` and `temp` addresses. A patient should have only a single current `billing` address. An address is considered 'valid' according to its period start and to dates.
-        *	if no current `billing` address is provided, a `temp` address may take precedence over the `home` address, again if it is valid according to its period start and to dates.
-        * if there is no valid, current `billing` and/or `temp` address, the `home` address must be used.
+        A home address is the patient's main residential address and should normally be used for all clinical and demographic purposes, including clinical and appointment correspondence. However additionally, temp and billing addresses may be provided by a patient when there is a requirement to record an alternative location for the purposes of unbroken care. When sending correspondence to a patient:
+        * a present and valid billing address may take precedence over home and temp addresses. A patient should have only a single current billing address. An address is considered 'valid' according to its period start and to dates.
+        * if no current billing address is provided, a temp address may take precedence over the home address, again if it is valid according to its period start and to dates.
```

Comment:
For me, the backticks, eg \`home\` help readability when rendered.
![example](https://github.com/NHSDigital/personal-demographics-service-api/assets/109090521/2ee28574-f021-4314-a7e2-304f6bc126b9)

Reply:
Thanks for flagging this - this should now be fixed 👍